code/datums/status_effects/debuffs.dm

```diff
@@ -16,6 +16,21 @@
 		duration = set_duration
 	return ..()
 
+//STAGGERED
+/datum/status_effect/incapacitating/stagger
+	id = "paralyzed"
+
+/datum/status_effect/incapacitating/stagger/on_apply()
+	. = ..()
+	if(!.)
+		return
+	ADD_TRAIT(owner, TRAIT_STAGGERED, TRAIT_STATUS_EFFECT(id))
+	SEND_SIGNAL(owner, COMSIG_LIVING_STAGGER_APPLIED, duration)
```

Comment:
Cant you use the signal from the trait being applied/removed instead?

Reply:
Replaced it with just modifying the scatter mod var, so cleaner code/less signals.